boto/connection.py

```diff
@@ -1106,6 +1106,15 @@ def _required_auth_capability(self):
         return []
 
 
+    def get_utf8_value(self, value):
+        """This replaces public interface get_utf8_value with new implementation.
+        
+        The old implementation of get_utf8_value has been deprecated and replaced
+        with get_utf8able_str.
```

Comment:
Let's not change the implementation out from underneath people (meaning that it would still exist but behave differently all of sudden -- uh oh).  Can we just make this call `boto.utils.get_utf8_value` again?

Reply:
Ah! Sure thing, I think I misread the prior comment. Will revert to the old implementation.